Refresh.GameServer/Types/Assets/AssetFlags.cs

```diff
@@ -0,0 +1,114 @@
+            GameAssetType.ThingRecording => method == GameSerializationMethod.Binary ? AssetFlags.None : AssetFlags.Modded,
+            GameAssetType.SyncedProfile => method == GameSerializationMethod.Binary ? AssetFlags.None : AssetFlags.Modded,
+            GameAssetType.GriefSongState => method == GameSerializationMethod.Binary ? AssetFlags.None : AssetFlags.Modded,
+            GameAssetType.Quest => method == GameSerializationMethod.Binary ? AssetFlags.None : AssetFlags.Modded,
+            GameAssetType.AdventureSharedData => method == GameSerializationMethod.Binary ? AssetFlags.None : AssetFlags.Modded,
+            GameAssetType.AdventureCreateProfile => method == GameSerializationMethod.Binary ? AssetFlags.None : AssetFlags.Modded,
+            
+            GameAssetType.VoiceRecording => method == GameSerializationMethod.Binary ? AssetFlags.Media : AssetFlags.Modded | AssetFlags.Media,
+            GameAssetType.Painting => method == GameSerializationMethod.Binary ? AssetFlags.Media : AssetFlags.Modded | AssetFlags.Media,
+            // Textures with the wrong serialization method will not be loadable
+            GameAssetType.Texture => method == GameSerializationMethod.CompressedTexture
+                ? AssetFlags.Media
+                : AssetFlags.Dangerous | AssetFlags.Modded | AssetFlags.Media,
+            GameAssetType.Jpeg => AssetFlags.Media,
+            GameAssetType.Png => AssetFlags.Media,
+            GameAssetType.Tga => AssetFlags.Media,
+            GameAssetType.Mip => AssetFlags.Media,
+            
+            // Textures with the wrong serialization method will not be loadable
+            GameAssetType.GameDataTexture => method == GameSerializationMethod.CompressedTexture
+                ? AssetFlags.Modded | AssetFlags.Media
+                : AssetFlags.Dangerous | AssetFlags.Media | AssetFlags.Modded,
+            GameAssetType.AnimatedTexture => method == GameSerializationMethod.Binary
+                ? AssetFlags.Modded | AssetFlags.Media
+                : AssetFlags.Dangerous | AssetFlags.Modded | AssetFlags.Media,
+            
+            GameAssetType.GfxMaterial => AssetFlags.Modded,
+            GameAssetType.Material => AssetFlags.Modded,
+            GameAssetType.Mesh => AssetFlags.Modded,
+            GameAssetType.Palette => AssetFlags.Modded,
+            GameAssetType.SoftPhysicsSettings => AssetFlags.Modded,
+            GameAssetType.Bevel => AssetFlags.Modded,
+            GameAssetType.Animation => AssetFlags.Modded,
+            GameAssetType.SettingsCharacter => AssetFlags.Modded,
+            GameAssetType.Joint => AssetFlags.Modded,
+            GameAssetType.MusicSettings => AssetFlags.Modded,
+            GameAssetType.StaticMesh => AssetFlags.Modded,
+            GameAssetType.PaintBrush => AssetFlags.Modded,
+            
+            GameAssetType.Script => AssetFlags.Dangerous | AssetFlags.Modded,
```

Comment:
Are there any cases of `Dangerous` without `Modded`? Makes the bitwise system in general feel a bit redundant.

Reply:
Discard this; meant to delete